repo-inventory/src/main/java/fr/lateb/presentation/rest/MenuResource.java

```diff
@@ -0,0 +1,52 @@
+package fr.lateb.presentation.rest;
+
+import fr.lateb.domain.entity.BeerEntity;
+import fr.lateb.domain.service.BeerService;
+import fr.lateb.util.BeerNotFoundError;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+@Tag(name = "Inventory", description = "Gestion des bières")
+@Path("/inventory")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class MenuResource {
+
+    @Inject
+    BeerService beerService;
+
+
+    @GET
+    @Path("/")
+    public Response getInventory() {
+        var beers = beerService.getAllBeers();
+        return Response.ok(beers).build();
+    }
+
+    @GET
+    @Path("/barcode") // Example : /inventory/barcode?id=456879
```

Comment:
```suggestion
    @Path("/beer") // Example : /inventory/beer?id=456879
```
We don't search a barcode but a beer